source/preprocessor.tex

```diff
@@ -1783,19 +1793,15 @@
 Otherwise, the original spelling of each preprocessing token in the
 stringizing argument is retained in the character string literal,
 except for special handling for producing the spelling of
-\grammarterm{string-literal}s and \grammarterm{character-literal}s:
-a
-\tcode{\textbackslash}
-character is inserted before each
-\tcode{"}
-and
-\tcode{\textbackslash}
-character of a \grammarterm{character-literal} or \grammarterm{string-literal}
-(including the delimiting
-\tcode{"}
-characters).
+\grammarterm{character-literal}s and \grammarterm{string-literal}s
```

Comment:
```suggestion
\grammarterm{character-literal}{s} and \grammarterm{string-literal}{s}
```

Reply:
I believe we established that this was an old convention that is no longer needed, and would be using the plain "s" plural going forward.

@jensmaurer, @tkoeppe do you remember the thread I am talking about?

Reply:
Yes, fine without braces, and it's also fine to not touch old text unnecessarily. This doesn't affect the presentation.